client/my-sites/earn/memberships/index.jsx

```diff
@@ -566,9 +638,13 @@ class MembershipsSection extends Component {
 			return this.renderOnboarding(
 				<Notice
 					status="is-warning"
-					text={ this.props.translate(
-						'Only site administrators can edit Recurring Payments settings.'
-					) }
+					text={
+						isEnabled( 'earn/rename-payment-blocks' )
+							? this.props.translate( 'Only site administrators can edit Payments settings.' )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.